### PR TITLE
Fix the scalapb json marshaller for sealed-oneof messages

### PR DESCRIPTION
**Pull request by andrew-teirney**
When returning a ScalaPB message of [the `sealed-oneof` flavour](https://scalapb.github.io/docs/sealed-oneofs/),
the JSON marshaller does not marshal the wrapping message.

This changeset corrects that by detecting the case where the value passed in
is a `GeneratedSealedOneof`, and then marshalls the container message.

### CODE REVIEW

File: scalapb/scalapb_2.13/src/main/scala/com/linecorp/armeria/common/scalapb/ScalaPbJsonMarshaller.scala [hunk 1]
```diff
@@ -39,13 +39,16 @@ final class ScalaPbJsonMarshaller private (
...
-    if (!message.isInstanceOf[GeneratedMessage])
-      throw new IllegalStateException(
-        s"Unexpected message type: ${message.getClass} (expected: ${classOf[GeneratedMessage]})")
-    val msg = message.asInstanceOf[GeneratedMessage]
-    os.write(jsonPrinter.print(msg).getBytes())
-  }
+  override def serializeMessage[A](marshaller: Marshaller[A], message: A, os: OutputStream): Unit =
+    message match {
+      case msg: GeneratedSealedOneof =>
+        os.write(jsonPrinter.print(msg.asMessage).getBytes())
```
Comment (ikhoon) [inline scalapb/scalapb_2.13/src/main/scala/com/linecorp/armeria/common/scalapb/ScalaPbJsonMarshaller.scala:45]: Awesome! Could you add tests for verifying that Armeria server and client correctly marshaling oneof?

**Comment by trustin** [comment]
@andrew-teirney Gentle ping

**Review by trustin — APPROVED** [review]
Thanks, @andrew-teirney and @ikhoon !

**Review by ikhoon — APPROVED** [review]
Thanks! @andrew-teirney

File: .appveyor.build.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-JRE8_URL='https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u242-b08/OpenJDK8U-jre_x64_linux_hotspot_8u242b08.tar.gz'
+JRE8_URL='https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u282-b08/OpenJDK8U-jre_x64_linux_hotspot_8u282b08.tar.gz'
 JRE8_VERSION='AdoptOpenJDK-8u242b08'
```
Comment (trustin) [inline .appveyor.build.sh:5]: Update this as well?

Comment (ikhoon) [reply .appveyor.build.sh:5]: Oops...

**Review by minwoox — APPROVED** [review]
Thanks, @andrew-teirney and @ikhoon !!!

**Pull request merged**